### DISCUSSION TITLE
warning pacote microdatasus

### DISCUSSION THREAD

**Discussion by carolinnastrindade** [rfsaldanha/microdatasus#71] [Q&A]
Atualmente estou desenvolvendo um estudo cuja base de dados é composta por informações do SIH e do SIM do DATASUS.

Para realizar o download dos dados diretamente do tabnet eu estou utilizando o pacote “microdatasus” criado pelo Dr. Raphael Saldanha.

Contudo, ao baixar no R as informações do SIH para os dados entre os anos de 2000 e 2007 referente aos estados que compõem a região nordeste do Brasil, o próprio R me retornou com o seguinte aviso:

The following dates (yymm) are from old folders and may contain incompatible codes (including old ICD codes):

0001, 0002, 0003, 0004, 0005, 0006, 0007, 0008, 0009, 0010, 0011, 0012 (para o ano de 2000)

0101, 0102, 0103, 0104, 0105, 0106, 0107, 0108, 0109, 0110, 0111, 0112 (para o ano de 2001)

0201, 0202, 0203, 0204, 0205, 0206, 0207, 0208, 0209, 0210, 0211, 0212 (para o ano de 2002)

0301, 0302, 0303, 0304, 0305, 0306, 0307, 0308, 0309, 0310, 0311, 0312 (para o ano de 2003)

0501, 0502, 0503, 0504, 0505, 0506, 0507, 0508, 0509, 0510, 0511, 0512 (para o ano de 2005)

0701, 0702, 0703, 0704, 0705, 0706, 0707, 0708, 0709, 0710, 0711, 0712 (para o ano de 2007)

Andei pesquisando e encontrei a informação de que a variável “DIAG_PRINC” é construída a partir da CID10.

Deste modo, não sei ao certo ao que este erro se refere. Será algo especificamente relacionado à questão da CID10?

**Comment by rfsaldanha** [comment]
Olá,

Coloquei esta mensagem no pacote pois o download dos arquivos está sendo realizado em pastas que podem conter arquivos que usem a CID-9. Deste modo, minha sugestão é avaliar cada ano para ter certeza de qual codificação está sendo adotada.
